inspirehep/utils/citations.py

```diff
@@ -42,8 +44,7 @@ def citations(self):
 
         out = []
         row = []
-        recid = self.record['control_number']
-        es_query = Query('refersto:' + str(recid)).search()
+        es_query = Query('refersto:' + str(self.recid)).search()
```

Comment:
For the references you can leave it with all of them.

Can you check how it is done now so that we get the number when the page loads?

![screen shot 2016-02-05 at 5 01 42 pm](https://cloud.githubusercontent.com/assets/580727/12851488/68421706-cc2a-11e5-93aa-22ecb5998d0e.png)

So either we leave it as it is now, or yes, receiving it on the AJAX request sounds good.